lms/djangoapps/certificates/signals.py

```diff
@@ -156,3 +159,18 @@ def _listen_for_enrollment_mode_change(sender, user, course_key, mode, **kwargs)
                 course_key,
             )
             return False
+
+
+@receiver(CERTIFICATE_CREATED)
+def listen_for_certificate_created_event(sender, signal, **kwargs):
+    """
+    Publish `CERTIFICATE_CREATED` events to the event bus.
+    """
+    if SEND_CERTIFICATE_CREATED_SIGNAL.is_enabled():
+        get_producer().send(
+            signal=CERTIFICATE_CREATED,
+            topic='certificate-created',
+            event_key_field='certificate.course.course_key',
```

Comment:
As I understand it, this `event key` is how we will partition data on the Kafka side and is required to ensure that multiple consumers can process events in order. I chose to go with `course_key` as this would potentially have fewer partitions than other possibilities (like user/user-id).

Reply:
Kafka will only give us as many partitions as we configure it to (right now the default is 6). All events with a certain key will only go to one partition, but one partition can (and almost certainly will) handle multiple keys. Similarly, a given partition can only be consumed by one consumer per group, but one consumer may handle multiple partitions.